recipes/ux.rb

```diff
@@ -36,62 +36,62 @@
 end
 
 log 'Checkout and Unzip the latest UX Code'
-remote_file '/home/jellyfish/ux-master.zip' do
+remote_file "#{node['rbenv']['user_home']}/ux-master.zip" do
   source 'https://github.com/projectjellyfish/ux/archive/master.zip'
   mode '0644'
 end
 
 log 'Unzip ux-master and move it to ux'
 bash 'unzip ux-master.zip' do
-  cwd '/home/jellyfish'
-  user 'jellyfish'
+  cwd node['rbenv']['user_home']
+  user node['jellyfish']['user']
   code <<-EOH
   unzip ux-master.zip
   EOH
-  creates '/home/jellyfish/ux-master'
+  creates "#{node['rbenv']['user_home']}/ux-master"
 end
 
 bash 'mv ux-master ux' do
```

Comment:
We should be able to remove this whole section by updating the unzip block above to use this code (will need some tweaking)

```
unzip ux-master.zip -d "/home/jellyfish/ux"
```